nameko/cli/commands.py

```diff
@@ -46,6 +46,31 @@ def main(args):
         main(args)
 
 
+class Config(Command):
+    """ Output as YAML string the configuration that would be passed to a
+    service.
+
+    Useful for viewing config files that load values from environement
+    variables.
+    """
+
+    name = 'config'
```

Comment:
how about calling this `show-config` or similar to make it more clearly distinct from `--config`

Reply:
That is a much better name!